include/rmm/exec_policy.hpp

```diff
@@ -72,8 +84,8 @@ class exec_policy_nosync : public thrust_exec_policy_nosync_t {
 
 #else
 
-using thrust_exec_policy_nosync_t = thrust_exec_policy_t;
-using exec_policy_nosync          = exec_policy;
+using thrust_exec_policy_nosync_t = thrust_exec_policy_t;  ///< See above
```

Comment:
I wish we could remove old Thrust support here, but it'd break rmm if used with the CUDA 11 toolkit versions of Thrust. Does that matter? If we think that's okay (and we can require rmm users to use thrust from rapids-cmake / CPM, via the conda CCCL packages, or some other means) then we could drop this macro conditional. Thoughts?

In the meantime, I'd document this differently than `See above`. Something like `When used with Thrust < 1.16, the nosync policy is an alias for the sync policy.`

Reply:
I agree a bit clearer doc is better here. I think it's OK to wait until CUDA 11 is sufficiently legacy to remove this -- it doesn't cost or hurt much to keep it around.

Reply:
I like Bradley's wording, I'll go with that.